demos/dialog/index.html

```diff
@@ -153,6 +155,29 @@
         Secondary
       </mwc-button>
     </mwc-dialog>
+
+    <mwc-button data-num="8" raised>Form Validation</mwc-button>
+    <mwc-dialog id="dialog8" heading="Form Validation">
+      <p>This dialog can validate user input before closing.</p>
+      <mwc-textfield
+        id="dialog8-text-field"
+        minlength="3"
+        maxlength="64"
+        placeholder="First Name"
+        required>
+      </mwc-textfield>
+      <mwc-button
+        id="dialog8-primary-action-button"
+        slot="primaryAction">
+        confirm
```

Comment:
Same nit here and for the cancel button. Even though these are Transformed to upper case, it's still a good habit to write them in sentence case: "Confirm" and "Cancel"

Reply:
yes, makes sense!